### PR TITLE
Support scripting in Debezium Server

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.redhat.com/browse/DBZ-5154

### CODE REVIEW

File: debezium-scripting/debezium-scripting-opt-depts/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-scripting-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+	<relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-scripting-opt-depts</artifactId>
```
Comment (jpechane) [inline debezium-scripting/debezium-scripting-opt-depts/pom.xml:10]: Please rename to `debezium-scripting-languages` and change the `name` below.

**Review by jpechane — CHANGES_REQUESTED** [review]
@vjuranek Nice wor, thanks. Please just rename the module with dependencies and I think we are good to go.

File: debezium-scripting/debezium-scripting-opt-depts/pom.xml (hunk 1 above)
Comment (vjuranek) [reply debezium-scripting/debezium-scripting-opt-depts/pom.xml:10]: done

**Comment by jpechane** [comment]
@vjuranek Tested and applied, thanks! COuld you please add one more PR with the same Jira adding the info into the documentation?

**Comment by vjuranek** [comment]
@jpechane sure, done in PR #3554

**Pull request merged**